libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Superfluous empty line at beginning of file.

Reply:
removed